core/templates/dev/head/pages/dashboard/dashboard.html

```diff
@@ -43,31 +44,45 @@ <h1 class="stat-value"><[dashboardStats.total_plays]></h1>
           <p class="stat-description">Open feedback</p>
           <h1 class="stat-value"><[dashboardStats.total_open_feedback]></h1>
         </div>
+        <div class="total-open-feedback stats-card">
+          <p class="stat-description">Subscribers</p>
+          <h1 class="stat-value"><[subscribersList.length]></h1>
+        </div>
       </md-card>
     </div>
 
-    <div ng-if="explorationsList.length > 0 || collectionsList.length > 0">
+    <div ng-if="explorationsList.length > 0 || collectionsList.length > 0 || subscribersList.length > 0">
```

Comment:
Hmm, no, I don't think so. If the user hasn't created any explorations then I think the only message on the page should be "start creating", i.e. the default 'new user' creator dashboard. Everything else is just a distraction.